content/docs/howto/negate-a-disjunction/en.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Negate a disjunction
+tags:
+- validation
+- language
+- commented cue
+authors:
+- hansbogert
+relatedIssues:
+- https://github.com/cue-lang/cue/discussions/3229
+- https://github.com/cue-lang/docs-and-content/issues/160
+toc_hide: true
+---
+
+This [Commented CUE]({{< relref "docs/howto#commented-cue-guides" >}})
+demonstrates how to negate a disjunction.
+
+{{{with code "en" "example"}}}
+! exec cue vet example.cue
+cmp stderr out
+-- example.cue --
+package example
+
+// Define a disjunction
```

Comment:
Fields/etc are "declared". However, rather than restating the operation of the following line (i.e. a relatively trivial comment explaining code that's pretty self-explanatory), I feel this comment would be most useful if it did double duty with something like:

```
// This disjunction permits a single value from a small set of options.
```

(but word-wrapped at character 51, as you've successfully discovered that we use, everywhere else!)

Also: we're trying to be consistent with CUE comments that are sentences (hence start with a capital unless it's impossible to do so) and end with a period.